src/nyctaxi/data_processor.py

```
@@ -0,0 +1,56 @@
+
+#from pyspark.sql import SparkSession
+from databricks.connect import DatabricksSession
+import pandas as pd
```

Ruff

`pandas` imported but unused

Remove unused import: `pandas`

(F401)